### DISCUSSION TITLE
Install SSL certificate?

### DISCUSSION THREAD

**Discussion by development-vargamarcel** [nhost/nhost#40] [Ideas]
Maybe it would be useful to be able to install an SSL certificate right from the dashboard. If possible.

**Comment by elitan** [comment]
To what service do you need this SSL certificate? What's the use case?

**Comment by mrinalwahal** [comment]
SSL certificates are useful more if you are deploying front-end services. Nhost only provides backend for now. And that is already deployed to a unique namespace for you in our server. Just use Cloudflare (or any other DNS provider which automatically installs free SSL certs) to map subdomains (example: `backend.xyz.com`) to your Nhost URLs?

You can find these settings for adding a custom domain under `DNS` tab in Nhost project settings.
